src/commands/step-functions/aws.ts

```diff
@@ -0,0 +1,154 @@
+import {CloudWatchLogs, StepFunctions} from 'aws-sdk'
+
+import {
+  CreateLogGroupRequest,
+  DeleteSubscriptionFilterRequest,
+  PutSubscriptionFilterRequest,
+  TagLogGroupRequest,
+  TagStepFunctionRequest,
+  UntagLogGroupRequest,
+  UpdateStepFunctionRequest,
+} from './interfaces'
+
+export const createLogGroup = (cloudWatchLogsClient: CloudWatchLogs, logGroupName: string): CreateLogGroupRequest => {
+  const params = {
+    logGroupName,
+    tags: {
+      DD_INSTRUMENTATION_SOURCE: 'datadog-ci',
+    },
+  }
+
+  return {
+    function: cloudWatchLogsClient.createLogGroup(params),
+    operation: 'createLogGroup',
```

Comment:
Could it make sense to have operation to be an enum instead of strings?

Reply:
Added an Operation enum for these values

```typescript
export enum Operation {
  createLogGroup = 'createLogGroup',
  deleteSubscriptionFilter = 'deleteSubscriptionFilter',
  putSubscriptionFilter = 'putSubscriptionFilter',
  tagLogGroup = 'tagLogGroup',
  tagResource = 'tagResource',
  untagLogGroup = 'untagLogGroup',
  updateStateMachine = 'updateStateMachine',
}
```

Reply:
Let's capitalize the first letter of the enum too!
https://www.typescriptlang.org/docs/handbook/enums.html
🙏🏽

Reply:
```
Enums are types, so they should be named using UpperCamelCase like classes. The enum values are constants, so they should be named using lowerCamelCase like constants, or ALL_CAPS if your code uses that legacy naming style.
```

Reply:
Thank you both! Here's the updated enum:

```typescript
export enum Operation {
  CreateLogGroup = 'createLogGroup',
  DeleteSubscriptionFilter = 'deleteSubscriptionFilter',
  PutSubscriptionFilter = 'putSubscriptionFilter',
  TagLogGroup = 'tagLogGroup',
  TagResource = 'tagResource',
  UntagLogGroup = 'untagLogGroup',
  UpdateStateMachine = 'updateStateMachine',
}
```

Reply:
While working on the unit tests I found a method to pull the `operation` and `params` from the AWS Request object. So that removed the need for this enum altogether.

https://github.com/DataDog/datadog-ci/blob/duncan-harvey/add-step-function-command/src/commands/step-functions/changes.ts#L15